Topic0Ejercicio3/.classpath

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
```

Comment:
Do not include classpath in pull requests. This is local enviroment specific configuration.